### ISSUE TITLE
All VideoScaleType / tviScaleType settings crop video

### ISSUE THREAD

**Issue by KamilSucharski** [twilio/video-quickstart-android#730]
### Description

No matter what scale type I select the video is always cropped. I also experimented with the view size (wrap_content, match_parent, static 300dp x 300dp square) and it was still cropped in all cases.

### Steps to Reproduce

1. Use the code below for view.
2. Connect to a room.

#### Code

```xml
    <androidx.constraintlayout.widget.ConstraintLayout
        android:layout_width="match_parent"
        android:layout_height="match_parent"
        android:background="@color/black">

        <com.twilio.video.VideoTextureView
            android:id="@+id/videoCallRemoteView"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            app:isVisible="@{viewModel.roomManager.isDoctorCameraEnabled}"
            app:layout_constraintBottom_toBottomOf="parent"
            app:layout_constraintEnd_toEndOf="parent"
            app:layout_constraintStart_toStartOf="parent"
            app:layout_constraintTop_toTopOf="parent"
            app:remoteVideoTrack="@{viewModel.roomManager.doctorVideoTrack}"
            app:tviScaleType="fit" />
```

#### Expected Behavior

Whole scene should be visible regardless of phone orientation. The view size should adjust, or at least black bars should be added to fill in the space.

#### Actual Behavior

The video is cropped:
![obraz](https://user-images.githubusercontent.com/15111215/197697454-b5f45dda-3d69-4fc2-85ea-9c97948ba4ed.png)

#### Reproduces how Often

Always.

#### Logs

Logs are not relevant here I think.

### Versions

All relevant version information for issue.

#### Video Android SDK

7.4.0

#### Android API

33

#### Android Device

Google Pixel 6a

**Comment by afalls-twilio** [comment]
@KamilSucharski thank you for reporting this issue. A ticket regarding this issue has been filed and while we are in the midst of a large release and as soon as that is accomplished we will begin addressing more issues (including this one). Thank you for your patience.

**Comment by afalls-twilio** [comment]
@m-sabeeh @margaritajankovska This issue has been fixed and will be in the next release (7.6.2+).
